librz/debug/p/native/reg/windows-arm64.h

```diff
@@ -1,108 +1,229 @@
+		"drx     bvr1    .64     832     0\n"
+		"drx     bvr2    .64     840     0\n"
+		"drx     bvr3    .64     848     0\n"
+		"drx     bvr4    .64     856     0\n"
+		"drx     bvr5    .64     864     0\n"
+		"drx     bvr6    .64     872     0\n"
+		"drx     bvr7    .64     880     0\n"
+		"drx     wcr0    .32     888     0\n"
+		"drx     wcr1    .32     892     0\n"
+		"drx     wvr0    .64     896     0\n"
+		"drx     wvr1    .64     904     0\n");
+} else {
+	return strdup(
+		"=PC     pc\n"
+		"=SP     sp\n"
+		"=BP     fp\n"
+		"=LR     lr\n"
+		"=A0     r0\n"
+		"=A1     r1\n"
+		"=A2     r2\n"
+		"=A3     r3\n"
+		"=ZF     zf\n"
+		"=SF     nf\n"
+		"=OF     vf\n"
+		"=CF     cf\n"
+		"flg     cpsr    .32     4       0   _____tfiae_____________j__qvczn\n"
+		"flg     vf      .1      4.28    0       overflow\n"
+		"flg     cf      .1      4.29    0       carry\n"
+		"flg     zf      .1      4.30    0       zero\n"
+		"flg     nf      .1      4.31    0       sign\n"
+		"gpr     r0      .32     8       0\n"
+		"gpr     r1      .32     16      0\n"
+		"gpr     r2      .32     24      0\n"
+		"gpr     r3      .32     32      0\n"
+		"gpr     r4      .32     40      0\n"
+		"gpr     r5      .32     48      0\n"
+		"gpr     r6      .32     56      0\n"
+		"gpr     r7      .32     64      0\n"
+		"gpr     r8      .32     72      0\n"
+		"gpr     r9      .32     80      0\n"
+		"gpr     r10     .32     88      0\n"
+		"gpr     fp      .32     96      0\n"
+		"gpr     ip      .32     104     0\n"
+		"gpr     sp      .32     112     0\n"
+		"gpr     lr      .32     120     0\n"
+		"gpr     pc      .64     264     0\n"
+		"fpu     v0      .128    272     0\n"
+		"fpu     v1      .128    288     0\n"
+		"fpu     v2      .128    304     0\n"
+		"fpu     v3      .128    320     0\n"
+		"fpu     v4      .128    336     0\n"
+		"fpu     v5      .128    352     0\n"
+		"fpu     v6      .128    368     0\n"
+		"fpu     v7      .128    384     0\n"
+		"fpu     v8      .128    400     0\n"
+		"fpu     v9      .128    416     0\n"
+		"fpu     v10     .128    432     0\n"
+		"fpu     v11     .128    448     0\n"
+		"fpu     v12     .128    464     0\n"
+		"fpu     v13     .128    480     0\n"
+		"fpu     v14     .128    496     0\n"
+		"fpu     v15     .128    512     0\n"
+		"fpu     v16     .128    528     0\n"
+		"fpu     v17     .128    544     0\n"
+		"fpu     v18     .128    560     0\n"
+		"fpu     v19     .128    576     0\n"
+		"fpu     v20     .128    592     0\n"
+		"fpu     v21     .128    608     0\n"
+		"fpu     v22     .128    624     0\n"
+		"fpu     v23     .128    640     0\n"
+		"fpu     v24     .128    656     0\n"
+		"fpu     v25     .128    672     0\n"
+		"fpu     v26     .128    688     0\n"
+		"fpu     v27     .128    704     0\n"
+		"fpu     v28     .128    720     0\n"
+		"fpu     v29     .128    736     0\n"
+		"fpu     v30     .128    752     0\n"
+		"fpu     v31     .128    768     0\n"
+		"gpr     fpcr    .32     784     0\n"
```

Comment:
Would be nice to move `fpcr` to `flg` with all of the bit setting, same as `cpsr`. Otherwise it should be in `fpu` since it's only relevant for floating point.